modules/builds-docker-credentials-private-registries.adoc

```diff
@@ -87,6 +79,15 @@ the build configuration:
 $ oc set build-secret --push bc/sample-build dockerhub
 ----
 
+. You can also link the push secret to the service account used by the build instead of specifying
+the `pushSecret` field. By default builds use the *builder* service account. The push secret is
+automatically added to the build if the secret contains a credential that matches the repository
+hosting the build's output image.
++
+----
+$ oc secrets link builder dockerhub
+----
+
```

Comment:
should the you must specify a from note be placed here as well ?

Reply:
I don't think so - you always need to specify the output image for push. If no output image is specified, image push doesn't happen 😄

Reply:
sounds good